common/telemetry.js

```diff
@@ -25,9 +25,17 @@ const networkInfoWriteClient = influx.getWriteApi(
   's' // precision
 )
 
+// Add new write client for batch metrics
+const batchMetricsWriteClient = influx.getWriteApi(
+  'Filecoin Station', // org
+  'spark-batch-metrics', // bucket
```

Comment:
```suggestion
  'spark-web3-storage-batch-metrics', // bucket
```

Maybe something like this would be more appropriate.